training/distributed_training/pytorch/model_parallel/gpt-j/11_train_gptj_smp_tensor_parallel_notebook.ipynb

```diff
@@ -0,0 +1,988 @@
+{
```

Comment:
<p>Nit: is this an extra "store" comment that can be removed? Or does the command print out the stored vars? Also, why is store necessary rather than setting variables?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>